Makefile

```diff
@@ -90,6 +92,16 @@ release/tag:
 	git tag -a $(VERSION) -m "Release $(VERSION)"
 	git push origin $(VERSION)
 
+## executes a dry run of the release process
+.PHONY: release/dry-run
+release/dry-run: 
+	go run github.com/goreleaser/goreleaser@$(GO_RELEASER_VERSION) release --snapshot --rm-dist
```

Comment:
in the PR description you said it was possible to check the dist directory but here you provide the `--rm-dist` option :thinking: 

```
--rm-dist                      Removes the dist folder
```

Reply:
hmm true, that is odd.. I double checked and even with that option the dist file persists :thinking: 
when you run the dry run, do you also get the dist folder after ?